<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LMForm</class>
 <widget class="QWidget" name="LMForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
border-style: solid;
border-width: 1px;
border-radius: 10px;
border-color: grey;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Landing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,0,0">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true">QGroupBox{
border-style: solid;
border-width: 1px;
border-radius: 10px;
border-color: transparent;
}</string>
        </property>
        <property name="title">
         <string>Sessions</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="10,1,1,1,1,1,1,1,10">
         <item row="1" column="0" rowspan="2">
          <widget class="QPushButton" name="requestLeadButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Request Lead</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" rowspan="2">
          <widget class="QComboBox" name="actionComboBox"/>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QPushButton" name="sendQuizButton">
           <property name="text">
            <string>Send Quiz</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="randomLocatorButton">
           <property name="text">
            <string>Random Locator</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="locatorLineEdit"/>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QPushButton" name="getAvailabilityButton">
           <property name="text">
            <string>Get Availability</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="styleSheet">
         <string notr="true">QGroupBox{
border-style: solid;
border-width: 1px;
border-radius: 10px;
border-color: transparent;
}</string>
        </property>
        <property name="title">
         <string>Inbound Video Pin</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" rowstretch="10,0,0,0,0,0">
         <item row="0" column="0" colspan="2">
          <spacer name="verticalSpacer_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0" colspan="2">
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" rowspan="2" colspan="2">
          <widget class="QPushButton" name="ivnewsessionButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>inbound video new session</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="ivflagButton">
           <property name="text">
            <string>inbound video flag</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Desktop</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9,11">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>User (Agent)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="loginButton">
           <property name="text">
            <string>Login</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="statusComboBox"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="vateamComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="checkVateamButton">
           <property name="text">
            <string>Check Vateam</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="getVateamsButton">
           <property name="text">
            <string>Get Vateams</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="logoutButton">
           <property name="text">
            <string>Logout</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="changeStatusButton">
           <property name="text">
            <string>Change Status</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="aliveButton">
           <property name="text">
            <string>Alive</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="aliveComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Session</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,0,0,0,0,0">
         <item row="7" column="1">
          <widget class="QPushButton" name="loadDBLeadButton">
           <property name="text">
            <string>Load DB lead</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="scheduleSessionButton">
           <property name="text">
            <string>Schedule Session</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Open Lead</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="createLeadButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Create Lead</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="openSessionButton">
           <property name="text">
            <string>Open Session</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Finish Lead</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="sessionTypeComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="freeSessionResourcesButton">
           <property name="text">
            <string>Free Session Resources</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="searchDBLeadutton">
           <property name="text">
            <string>Search DB lead</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="setResultButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Set Result</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="getResultsButton">
           <property name="text">
            <string>Get Results</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
